src/tests/pull/project-state.json

```diff
@@ -1,49 +1,35 @@
 {
+  "allBranches": {
```

Comment:
Tu je potrebna zmena (este sa to stale nepouziva, projekt sa v tomto PR iba vycistí).
... Ak vytvorim nejake konfiguracie pred vytvorenim `dev-branch`, .... tak sa tieto konfiguracie "nakopiruju" do `dev-branch` ale maju rovnake ID (a daju sa mernut spat) .... odlisna situacia je ked vytvorim konfiguraciu iba v `main/dev branch` ... v ostatnych vetvach nebude. 

Preto som pridal kluc `allBranchces`, ktory vytvori konfiguracie v `default-branch` predtym ako sa vytvoria vetvy a konfiguracie v nich.